src/js/components/reviewData/ReviewDataContent.jsx

```diff
@@ -14,6 +14,8 @@ import RevalidateDataModal from './CertificationModal/RevalidateDataModal';
 import ReviewDataNarrative from './ReviewDataNarrative';
 import * as Icons from '../SharedComponents/icons/Icons';
 
+import * as UtilHelper from '../../helpers/util';
```

Comment:
You can also just import the function we need:
`import { formatSize } from ...`